ml/jml/probabilizer.h

```diff
@@ -332,6 +336,31 @@ operator << (DB::Store_Writer & store, const GLZ_Probabilizer & prob);
 DB::Store_Reader &
 operator >> (DB::Store_Reader & store, GLZ_Probabilizer & prob);
 
+/*****************************************************************************/
+/* PROBABILIZER                                                          */
+/*****************************************************************************/
```

Comment:
What is this class doing?  It's a factoring out of the actual probabilizer data parameters?  Then it should be POD, have no logger, and have no constructor.